WeakAurasOptions/OptionsFrames/ModelPicker.lua

```diff
@@ -220,7 +220,7 @@ local function ConstructModelPicker(frame)
   group:AddChild(modelTree);
 
   local model = CreateFrame("PlayerModel", nil, group.content);
-  model.SetTransformFixed = model.GetResizeBounds and OptionsPrivate.Private.ModelSetTransformFixed or model.SetTransform -- TODO change test to WeakAuras.IsWrathOrRetail() after 3.4.1 release
+  model.SetTransformFixed = model.GetResizeBounds and OptionsPrivate.Private.ModelSetTransformFixed or model.SetTransform -- TODO change test to WeakAuras.IsWrathOrCataOrRetail() after 3.4.1 release
```

Comment:
@mrbuds what was the context here?

Reply:
This was changed when new model API from Retail was retrofitted to wotlk in the middle of an extension ("classic_ptr" had different api than "classic")

This test can be remove now, all versions of the game including sod are using the same API